.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: Build & publish module to registry 
+
+on:
+  push:
+    tags:
+    - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
edit this to support -rc tags as well, e.g. `0.0.5-rc` used for testing and candidate purposes.

Reply:
[0-9]+.[0-9]+.[0-9]+(-rc)? I think is the regex for that

Reply:
done!